exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -40,6 +40,29 @@
    */
   HashCode submitTransaction(TransactionMessage tx);
 
+  /**
+   * Returns a number of unconfirmed transactions those are currently located in
+   * the memory pool and are waiting for acceptance to a block.
+   * @throws RuntimeException if the client is unable to complete a request
+   *        (e.g., in case of connectivity problems)
+   */
+  int getUnconfirmedTransactions();
```

Comment:
Will such a name work well when we add `getTransaction`/`getTransactions`, as it does not return transactions, but their number? `getNumUnconfirmedTransactions` — will be more accurate, but I am not quite happy with the name length :-)